pkg/logs/input/windowsevent/tailer.go

```diff
@@ -119,7 +119,7 @@ func (t *Tailer) toMessage(event string) (*message.Message, error) {
 	}
 	jsonEvent = replaceTextKeyToValue(jsonEvent)
 	log.Debug("Sending JSON:", string(jsonEvent))
-	return message.NewMessage(jsonEvent, message.NewOrigin(t.source), message.StatusInfo), nil
+	return message.NewPartialMessage2(jsonEvent, message.StatusInfo, t.source), nil
```

Comment:
What's wrong with the current constructor ?

Reply:
I am renaming the current constructor, it's taking incomplete list of parameters to construct an instance.

Reply:
I think it's fine to be honest, plus the name of the constructor is not really explicit